msobooking.yaml

```
@@ -0,0 +1,436 @@
+entityDefinitions:
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "MSOJailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SYSID"
+      ol.class:
+        type: "ol.class"
+        column: "CURRENT_CLASS"
+      ol.releasedatetime:
+        type: "ol.releasedatetime"
+        column: "RELDATE"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      criminaljustice.releasetype:
+        type: "criminaljustice.releasetype"
+        column: "RELTYPE"
+      ol.admissiontype:
+        type: "ol.admissiontype"
+        column: "ADTYPE"
+      ol.commitdatetime:
+        type: "ol.commitdatetime"
+        column: "COMDATE"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+    name: "jailstay"
+
+  person:
+    fqn: "general.person"
+    entitySetName: "MSOPerson"
```

```suggestion
    entitySetName: "MSOInmates"
```
let's be a bit more specific